src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/GtkTextBoxView.cs

```diff
@@ -90,6 +92,27 @@ public virtual void UpdateProperties(TextBox textBox)
 
 	public void SetFocus() => InputWidget.HasFocus = true;
 
+	public void SetClip(int clipLeft, int clipRight, int clipTop, int clipBottom)
```

Comment:
Is clipping the whole story? I think it `TextBox` could be just changing its placement without the native entry knowing about it.

Reply:
Note that currently we call `InvalidateLayout` on `StartEntry` only:

https://github.com/unoplatform/uno/blob/0938f2455bee51ba959e9a73b1ee404207550083/src/Uno.UI/UI/Xaml/Controls/TextBox/OverlayTextBoxView/OverlayTextBoxViewExtension.skia.cs#L54C3-L54C19

which looks part of the real bug from the linked issue.